### PR TITLE
Renew SDN infrastructure nodes certificates

### PR DESCRIPTION
**Pull request by ManikaDhiman**

### CODE REVIEW

**Comment by Court72** [comment]
@sethmanheim 

After this PR is out of draft state, can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-stack/hci/manage/update-sdn-infrastructure-nodes-certificates.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+# Renew certificates for Software Defined Networking infrastructure nodes
+
+> Applies to: Azure Stack HCI, versions 22H2 and 21H2; Windows Server 2022 and Windows Server 2019
+
+This article provides instructions on how to renew or change Software Defined Networking (SDN) server and Software Load Balancer (SLB) multiplexer (MUX) certificates. If you face any issues in renewing your certificates, contact Microsoft Support.
+
+For information about how to renew Network Controller certificates, see [Renew Network Controller certificates before they expire](./update-network-controller-certificates.md).
+
+In your SDN infrastructure, the Network Controller uses certificate-based authentication to secure Southbound communications with network devices, such as the SLB and the physical hosts. The certificates for the SLB and server come with a validity period, after which they become invalid and can no longer be trusted for use. You must renew them before they expire.
```
Comment (sethmanheim) [inline azure-stack/hci/manage/update-sdn-infrastructure-nodes-certificates.md:18]:
```suggestion
In your SDN infrastructure, the Network Controller uses certificate-based authentication to secure southbound communications with network devices, such as the SLB and the physical hosts. The certificates for the SLB and server come with a validity period, after which they become invalid and can no longer be trusted for use. You must renew them before they expire.
```

File: azure-stack/hci/manage/update-sdn-infrastructure-nodes-certificates.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+   - To generate self-signed certificates with a specific validity period, use the `NotAfter` parameter to specify the validity period. For example, to generate self-signed certificates with a validity period of five years, run the following commands:
+
+        ```powershell
+        Import-Module -Name SdnDiagnostics -Force
+        Start-SdnServerCertificateRotation -GenerateCertificate -CertPassword (Get-Credential).Password -NotAfter (Get-Date).AddYears(5) -Credential (Get-Credential)
+        ```
+
+1. After you confirm to continue with the certificate rotation, you can view the status of the ongoing operations in the PowerShell command window.
+
+   > [!Important]
```
Comment (sethmanheim) [inline azure-stack/hci/manage/update-sdn-infrastructure-nodes-certificates.md:86]:
```suggestion
   > [!IMPORTANT]
```

**Review by sethmanheim — APPROVED** [review]

**Comment by Jak-MS** [comment]
@ManikaDhiman - take this out of draft mode when it's ready for review and merge. thanks!

**Pull request merged**